pep_sphinx_extensions/pep_zero_generator/parser.py

```diff
@@ -22,6 +23,14 @@
     from pep_sphinx_extensions.pep_zero_generator.author import Author
 
 
+# AUTHOR_OVERRIDES.csv is an exception file for PEP0 name parsing
+AUTHOR_OVERRIDES: dict[str, dict[str, str]] = {}
+with open("AUTHOR_OVERRIDES.csv", "r", encoding="utf-8") as f:
```

Comment:
`"r"` is the default:
 
```suggestion
with open("AUTHOR_OVERRIDES.csv", encoding="utf-8") as f:
```

Reply:
I prefer including `'r'` for explicitness - I see your point but I don't think it hurts.

A